test_pyvodb/test_load.py

```diff
@@ -213,6 +213,7 @@ def test_talk_links(db):
     assert {s.kind: s.url for s in talk.links} == {
         'talk': 'http://lanyrd.com/2013/brnenske-pyvo-brug-kvetnove/schxdm/',
         'video': 'http://www.youtube.com/watch?v=HDmCGUKfe7Y',
+        'video': 'https://youtu.be/HDmCGUKfe7Y',
```

Comment:
Dict with two equal keys - please remove one, it's confusing

Reply:
Oops, how can this even work?! Thanks for spotting that, I'm pushing a fix.